docs/resources/supported-chains.md

```diff
@@ -18,15 +18,14 @@ You can learn more about compatible chains [here.](http://ethanfast.com/top-cryp
 ## Supported Chains (Access Control & PKP)
```

Comment:
Somewhere above or below this (maybe at the end of the "Programmable Key Pairs" section), let's add a note like this: "To sign on any chain not currently supported by Programmable Key Pairs, see "Wrapped Keys" [link to Wrapped Keys overview doc]."

Reply:
Note added:
![image](https://github.com/user-attachments/assets/8390fb9f-d91b-4f78-ac06-31f297940202)